x-pack/plugins/ingest_pipelines/public/application/components/pipeline_processors_editor/components/test_pipeline/test_pipeline_actions.tsx

```diff
@@ -49,6 +49,12 @@ export const TestPipelineActions: FunctionComponent = () => {
     });
   };
 
+  const stopPipelineSimulation = () => {
+    setCurrentTestPipelineData({
```

Comment:
nit; I think a better name for `setCurrentTestPipelineData` would be `testPipelineDataDispatch` (if we want to disambiguate) otherwise `dispatch`.